src/Pangraph/VHDL/GraphWriter.hs

```diff
@@ -4,23 +4,35 @@ module Pangraph.VHDL.GraphWriter (
 writeGraphVhdl
 ) where
 
+import Data.Maybe
+
 import qualified Pangraph         as P
-import qualified Data.ByteString as BS
-import Data.ByteString.Char8 (unpack)
 
+import Data.ByteString.Char8      (unpack)
+
+
+type VertexName  = String
+type VertexIndex = Int
+
+-- The following two functions are written so that this file does not have to be rewritten to handle cases of Maybe
+
+vertexValuesByKey :: P.Vertex -> P.Key -> [P.Value]
+vertexValuesByKey k v = maybeToList $ P.lookupVertexValues v k
+
+edgeValuesByKey :: P.Edge -> P.Key -> [P.Value]
+edgeValuesByKey k e = maybeToList $ P.lookupEdgeValues e k
 
-type NodeName  = String
-type NodeIndex = Int
+-- | Writes a Pangraph to VHDL
 
 writeGraphVhdl :: P.Pangraph -> String
```

Comment:
I have opted to name these `writeGraph` and `writeEnviroment` and re-export these in `Pangraph.VHDL` which saves the user from using two imports. I also pack these into `ByteString`s to save breaking changes later.